basic/gateway/src/main/resources/application.yaml

```diff
@@ -7,10 +7,14 @@ spring:
           uri: lb://basic-consumer
           predicates:
             - Path=/consumer/**,/benchmark/**
+          filters:
+            - StripPrefix=1
         - id: provider
           uri: lb://basic-provider
           predicates:
             - Path=/provider/**
+          filters:
+            - StripPrefix=1
 
       httpclient:
```

Comment:
Do you run any other test cases like bench mark? This change will broken other cases.

Reply:
fixed